### PR TITLE
Theming Adjustments

### PR DESCRIPTION
**Pull request by samgutentag**
Continued efforts to cleanup and simplify docs theming.

### CODE REVIEW

**Comment by samgutentag** [comment]
latest additions correct dark mode css selections and forces text to be black, much more readible.

File: src/css/custom.css [hunk 1]
```diff
@@ -5,6 +5,8 @@
  * work well for content-centric websites.
  */
 
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
```
Comment (jthiller) [inline src/css/custom.css]: per https://github.com/helium/docs/pull/1108, this should be removed and in the head instead

Comment (samgutentag) [reply src/css/custom.css]: the joys of github squishing... fixed!

**Comment by jthiller** [comment]
Can we just do a clean branch with this 1-line change? It makes me uneasy seeing that big stack of commits

**Comment by samgutentag** [comment]
> Can we just do a clean branch with this 1-line change? It makes me uneasy seeing that big stack of commits

dang it... committed fix before seeing your comment. agreed, this PR is getting to be a mess... let me take a look and submit a clean branch with the fixes we actually want

**Comment by samgutentag** [comment]
closing PR, clean branch fixes will be submitted in new PR

**Pull request closed without merging**